lib/createWorkers.js

```diff
@@ -48,10 +50,37 @@ function createWorkers(handlers, config, options = {}) {
 
   return {
     listen: co.wrap(_listen),
-    close: co.wrap(_close)
+    close: co.wrap(_close),
+    get emitter() {
+      return emitter;
+    },
+    wait(key) {
```

Comment:
From my point of view, this is the standard use case as an event is never guaranteed. As for a HTTP timeout, we have to implement it to not have blocked processes.